motion/reference_model/dp_reference_model/config/reference_model.yaml

```diff
@@ -1,10 +1,20 @@
 # Parameters used for the DP-reference model
 dp_rm:
+  zeta_1: 1.5
+  zeta_2: 1.5
+  zeta_3: 1.5
+  zeta_4: 1.0
+  zeta_5: 1.0
+  zeta_6: 1.0
+  zeta_7: 1.0
 
-  a1: 1
-  a2: -1.990024937655860
-  a3: 0.990049813123053
+  omega_1: 1.0
+  omega_2: 1.0
+  omega_3: 1.0
+  omega_4: 1.0
+  omega_5: 1.0
+  omega_6: 1.0
+  omega_7: 1.0
```

Comment:
Would it be possible to define these as arrays in the yaml and parse them as separate values instead of this?

Reply:
This was done so the parameters could be reconfigurable with dynamic reconfigure, as it is not possible to change the values of an array with it.

A solution to "streamline" this file would be to make it into array in this file, and then parametrize as individual values in the node, but I believe it would make it more confusing than necessary.

Although the reason for not array-ifing these params could commented in this file.